packages/syft-cli/cli.py

```diff
@@ -0,0 +1,29 @@
+# third party
+from rich import box
+from rich.console import Console
+from rich.table import Table
+import typer
+from version import __version__
+
+app = typer.Typer(name="Syft CLI", no_args_is_help=True)
+console = Console()
+
+
+@app.command()
+def hello():
+    typer.echo("Syft CLI 👋")
+
+
+@app.command()
+def version():
+    # syft absolute
+    import syft
```

Comment:
MAYBE: We could put importing syft in a try catch, for the usecases, in which use the CLI independently without installing syft to prevent the errors

Reply:
I am also thinking, on should we even display the syft version , when printing cli version ,as we are thinking of cli as an independent tool.

Reply:
Printing syft version also is okay , it should be not be an issue.

Reply:
> We could put importing syft in a try catch
Ah great idea - implemented this